onert-micro/onert-micro/include/pal/common/PALSoftmaxInputGrad.h

```diff
@@ -37,20 +37,25 @@ void inline SoftmaxInputGrad(const float *dloss_doutput_data,
 {
   assert(dloss_doutput_shape.dimensionsCount() == 2);
   assert(dloss_doutput_shape.dims(0) == 1);
-  const uint32_t output_dim = dloss_doutput_shape.dims(dloss_doutput_shape.dimensionsCount() - 1);
-  for (int i = 0; i < output_dim; ++i)
+  const uint32_t width = dloss_doutput_shape.dims(dloss_doutput_shape.dimensionsCount() - 1);
```

Comment:
@chunseoklee 
Just in case.
Is it in `NCHW`? Line 38 assures that it has only 2 dimensions. May be we can consider `last_dim` or previous name?
It's just minor note, so please ignore it.

Reply:
I will borrow this code from https://github.com/Samsung/ONE/blob/master/compute/cker/include/cker/train/operation/SoftMax.h. Let's keep it as it is for consistence.

Reply:
@chunseoklee 
Got it. Tnank you.